resources/zib/zib-TextResult-Media.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+    <id value="zib-TextResult-Media"/>
+    <url value="http://nictiz.nl/fhir/StructureDefinition/zib-TextResult-Media"/>
+    <name value="ZibTextResultMedia"/>
+    <title value="zib TextResult Media"/>
+    <status value="draft"/>
+    <publisher value="Nictiz"/>
+    <contact>
+        <name value="Nictiz"/>
+        <telecom>
+            <system value="email"/>
+            <value value="info@nictiz.nl"/>
+            <use value="work"/>
+        </telecom>
+    </contact>
+    <description value="Textual results describe the findings and interpretation of diagnostic or therapeutic procedures carried out on patients or on specimens from the patient."/>
```

Comment:
This comment doesn't make much sense for such a profile. Guess we'll need to be more specific in the profiling guidelines. See #66 (mainly a placeholder right now).